lib/assert.js

```diff
@@ -359,13 +359,13 @@ function _throws(shouldThrow, block, expected, message) {
 // 11. Expected to throw an error:
 // assert.throws(block, Error_opt, message_opt);
 
-assert.throws = function(block, /*optional*/error, /*optional*/message) {
+assert.throws = function throws(block, /*optional*/error, /*optional*/message) {
   _throws(true, block, error, message);
 };
 
 // EXTENSION! This is annoying to write outside this module.
-assert.doesNotThrow = function(block, /*optional*/error, /*optional*/message) {
+assert.doesNotThrow = function doesNotThrow(block, /*optional*/error, /*optional*/message) {
```

Comment:
@maasencioh I think the linter is not happy with this line as it is too long.
Can you please fix this? Thank you.